src/exceptions/ExeptionHelperClass.java

```diff
@@ -0,0 +1,55 @@
+package exceptions;
+
+import javax.naming.AuthenticationException;
+import java.io.FileNotFoundException;
+import java.net.BindException;
+
+public class ExeptionHelperClass {
+
+    static public void testExceptions()throws RuntimeException{
```

Comment:
```suggestion
    public static void testExceptions() throws RuntimeException {
```
access modifiers first, then static then final